2024/Autumn/Linux-Install-Fest/README.md

```diff
@@ -0,0 +1,10 @@
+# Linux Install Fest 2024
+
```

Comment:
Include info such as date, time, venue etc. and link to the design repo folder of its poster. Add the poster to the design repo if not done yet.

Reply:
Why? Even if it set a bad example, it should be archived imo (with a footnote or whatever).

Reply:
i approve otherwise